src/modules/terminalfont.c

```diff
@@ -179,6 +179,40 @@ static void printXCFETerminal(FFinstance* instance)
     }
 }
 
+static void printAlacritty(FFinstance* instance) {
+    FFstrbuf fontName;
+    FFstrbuf fontSize;
+    ffStrbufInit(&fontName);
+    ffStrbufInit(&fontSize);
+
+    FFpropquery fontQuery[] = {
+        {"family:", &fontName},
+        {"size:", &fontSize},
+    };
+
+    // alacritty parses config files in this order
+    ffParsePropFileConfigValues(instance, "alacritty/alacritty.yml", 2, fontQuery);
+    if(fontName.length == 0 || fontSize.length == 0)
+        ffParsePropFileConfigValues(instance, "alacritty.yml", 2, fontQuery);
+    if(fontName.length == 0 || fontSize.length == 0)
+        ffParsePropFileConfigValues(instance, ".alacritty.yml", 2, fontQuery);
+    if(fontName.length == 0 || fontSize.length == 0)
+    {
+         //by default alacritty uses it's own font called alacritty at size 11
+        ffStrbufAppendS(&fontName, "alacritty");
+        ffStrbufAppendS(&fontSize, "11");
+    }
```

Comment:
We should check for font and size independently, for example if font found, and size wasn't, this would override the font.

Reply:
good point